src/__main__.py

```diff
@@ -29,6 +36,14 @@ def install_debt_report_dependencies():
     )
 
 
+def get_dates_by_day_interval(start_date, end_date, interval_in_days):
+    dates_by_day_interval = []
```

Comment:
Comprehensión de lista.

Reply:
Cual seria recomendable usar?? Ya que segun [este](https://www.geeksforgeeks.org/declare-an-empty-list-in-python/) artículo, se puede declarar una lista vacia con [] o list()